experimenter/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -222,7 +223,18 @@ export const AppLayoutSidebarLaunched = ({
                   ) : analysis?.metadata?.external_config?.skip ? (
                     "Experiment analysis was skipped"
                   ) : (
-                    "Experiment analysis not ready yet"
+                    <>
+                      Experiment analysis not ready yet.
+                      {experiment.resultsExpectedDate && (
+                        <>
+                          {" "}
+                          Results are expected 8 days after enrollment ends{" "}
+                          (currently{" "}
+                          <b>{humanDate(experiment.resultsExpectedDate)}</b>
```

Comment:
This confused me, it wasn't clear if it meant the date was the enrollment end date or the results expected date.  I'd just take out the bit about '8 days after enrollment' and just say 'results are expected on <date>'

Reply:
Yep that makes sense, but let's just try a) and if we get feedback on it to add more detail about why it's that date we can expand it 👍